js/check.js

```diff
@@ -0,0 +1,47 @@
+import showError from "./showError.js";
+
+export const checkEmpty = (tag) => {
+  if(tag.value === '') {
+    if(tag.id === 'email'){
+      showError(tag, '이메일을 입력해주세요.');
+      return false;
+    }
+    if(tag.id === 'password') {
+      showError(tag, '비밀번호를 입력해주세요.');
+      return false;
+    }
+    if(tag.id === 'password-check') {
+      showError(tag, '비밀번호를 입력해주세요.');
+      return false;
+    }
+  }
+  return true;
+}
+
+export const checkEmail = (tag) => {
+  if(tag.value === '') return;
```

Comment:
일정하고 단순한 반환타입을 갖는게 좋습니다. 여기서는 boolean 값을 반환하는게 좋겠습니다
```suggestion
  if(tag.value === '') return false
```